Danki2/Assets/Scripts/AI/Editor/AIComponentEditor.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using UnityEditor;
+using UnityEngine;
+
+[CustomEditor(typeof(AIComponent))]
+public class AIComponentEditor : Editor
+{
+    public override void OnInspectorGUI()
+    {
+        base.OnInspectorGUI();
+
+        if (GUILayout.Button("Log Attribute Values"))
+        {
+            BehaviourScanner.Scan();
+            LogBehaviourData();
+        }
+    }
+
+    private void LogBehaviourData()
+    {
+        foreach (AIAction action in Enum.GetValues(typeof(AIAction)))
+        {
+            var data = BehaviourScanner.GetDataByAction(action);
+            string message = action.ToString() + " behaviours found: " + data.Count;
+            if (data.Count > 0) message += ": ";
+            foreach (var d in data)
+            {
+                message += d.DisplayValue + ", ";
+            }
+            message.Remove(message.Length - 2);
+
+            Debug.Log(message);
```

Comment:
I don't know if this is intended as production code, but I would use a bit of linq and string interpolation here, and replace lines 24-32 with something like:
```c#
var message1 = $"{action.ToString()} behaviours found: {data.Count}";
var message2 = string.Join(", ", data.Select(d => $"{d.DisplayValue}"));

Debug.Log(data.Count == 0 ? message1 : $"{message1}: {message2}"); 
```

Reply:
I've approved in any case, if you like the change you can pop it on the next PR.